packages/web/src/_page-tests/validators/explore.test.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react'
 import * as renderer from 'react-test-renderer'
 import { TestProvider } from 'src/_page-tests/test-utils'
 import ValidatorsList from 'src/dev/ValidatorsList'
+import { cleanData, ValidatorsListData } from 'src/utils/validators'
 
 const mock = {
```

Comment:
for instnace `const data: ValidatorsListData = ...`

Reply:
Fair enough, that was kind of lazy 😅